src/Datadog.Trace/PlatformHelpers/AzureAppServices.cs

```diff
@@ -77,6 +77,11 @@ internal class AzureAppServices
         /// </summary>
         internal const string OperatingSystemKey = "WEBSITE_OS";
 
+        /// <summary>
+        /// The unique name of the web site.
+        /// </summary>
+        internal const string WebsiteSiteNameKey = "WEBSITE_SITE_NAME";
```

Comment:
Doesn't appear to be used anywhere?

Reply:
I think I included this in a bad merge, thanks. I'll remove it.